src/systems/character-controller-system.js

```diff
@@ -124,10 +124,8 @@ export class CharacterControllerSystem {
         inPosition.setFromMatrixPosition(inMat4Copy);
         this.findPositionOnNavMesh(inPosition, inPosition, outPosition, true);
         finalPOV.setPosition(outPosition);
-        translation.makeTranslation(0, getCurrentPlayerHeight(), -0.15);
-      } else {
-        translation.makeTranslation(0, 1.6, -0.15);
       }
+      translation.makeTranslation(0, getCurrentPlayerHeight(), -0.15);
```

Comment:
Player height at spawn point is too big, but if we don't use it we spawn under the floor. Investigating the underlying issue here: https://github.com/mozilla/hubs/issues/2225

Reply:
Underlying issue resolved by moving to spawn point after entering vr. https://github.com/mozilla/hubs/pull/2226/commits/e5f6342819548435fc8998593771dd415f071b90